src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -115,22 +115,34 @@ CHIP_ERROR ScenesServer::Init()
 
     for (auto endpoint : EnabledEndpointsWithServerCluster(Id))
     {
-        // Explicit AttributeValuePairs is mandatory for matter so we force it here, ScenesName is not but it is forced for now
-        // TODO: We currently force SceneNames on but this needs to be modified to read the value generated from Zap instead.
-        uint32_t featureMap  = to_underlying(Feature::kExplicit) | to_underlying(Feature::kSceneNames);
-        EmberAfStatus status = Attributes::FeatureMap::Set(endpoint, featureMap);
+        uint32_t featureMap;
```

Comment:
We should assign a default value since we don't error out if the read fails. This avoids "undetermined" behavior letter in the code.

Reply:
The read of the feature map shouldn't fail and action should be taken there if it does no?

Reply:
It shouldn't fail; this is just to be consistent with the other changes where the `featureMap` has a default assignment.

Reply:
Default value added.

Reply:
Added initial value of 0.